hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/ConfigFactory.java

```diff
@@ -1,13 +1,22 @@
 package pl.allegro.tech.hermes.common.config;
 
+import com.netflix.config.AbstractPollingScheduler;
+import com.netflix.config.ConfigurationManager;
+import com.netflix.config.DynamicConfiguration;
 import com.netflix.config.DynamicPropertyFactory;
+import com.netflix.config.PolledConfigurationSource;
+import com.netflix.config.sources.URLConfigurationSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import static java.util.Arrays.stream;
 import static java.util.stream.Collectors.joining;
 
 public class ConfigFactory {
 
-    private final DynamicPropertyFactory propertyFactory = DynamicPropertyFactory.getInstance();
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConfigFactory.class);
```

Comment:
LOGGER -> logger, for please :)
(I know it's static, but it's not a constant, an enum, it's just a regular object that we use)

Reply:
such a habit

Reply:
;D